secretmanager/snippets/quickstart.py

```diff
@@ -65,4 +65,4 @@ def quickstart(_project_id=None, _secret_id=None):
 
 
 if __name__ == "__main__":
-    quickstart()
+    quickstart("YOUR_PROJECT_ID", "YOUR_SECRET_ID")
```

Comment:
I had issues trying to use `str | None` and have valid mypy tests, so I found just providing defaults here helped. I don't _think_ this quickstart is called directly, but we'll see if tests pass 🤞

Reply:
I had something similar with defaults. I had to change some defaults from None to "falsy" values of other types, like "", {}, []... This looks really good (and defaulting them to None doesn't seem to have been a good idea anyway).